subjects/java/checkpoints/distinct-substring-lenght/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Instructions
 
-Create a class `DistinctSubstringLength` that provides a method to find the length of the longest substring without repeating characters in a given string.
+Create a class `DistinctSubstringLength` that provides a method to find the length of the longest substring without repeating characters in case sensitive withen the given string.
```

Comment:
I would keep the first version of the sentence and add a short additional sentence (or an example) to show that the distinctions between characters should be case-sensitive.

An example of a sentence to be added:
```
The distinction between characters should be case-sensitive.
```

Reply:
done